x-pack/plugins/maps/public/routing/state_syncing/app_state_manager.ts

```diff
@@ -5,15 +5,24 @@
  */
 
 import { Subject } from 'rxjs';
+import { Filter, SavedQuery, Query } from 'src/plugins/data/public';
 
 export class AppStateManager {
-  _query = '';
-  _savedQuery = '';
-  _filters = [];
+  _query: string | { [key: string]: any } = '';
+  _savedQuery: SavedQuery | string = '';
+  _filters: Filter[] = [];
 
   _updated$ = new Subject();
 
-  setQueryAndFilters({ query, savedQuery, filters }) {
+  setQueryAndFilters({
+    query,
+    savedQuery,
+    filters,
+  }: {
+    query?: Query;
+    filters?: Filter[];
+    savedQuery?: SavedQuery | string;
```

Comment:
Should just be `savedQuery?: SavedQuery;`

Reply:
This was changed to `savedQueryId?: string`.